packages/astro/components/ViewTransitions.astro

```diff
@@ -233,15 +232,17 @@ const { fallback = 'animate' } = Astro.props as Props;
 			location.href = href;
 			return;
 		}
+		document.documentElement.dataset.astroTransition = dir;
```

Comment:
why here and not inside of updateDOM?

Reply:
:) 
1. symmetry: removing the attribute is done a few lines further down in "Navigate". Same level.
1. When I removed `doc.documentElement.dataset.astroTransition = dir` from `updateDOM`, it was clear that updating the DOM does not depend on a direction. (this was the only usage of `dir` in this function)
1. simpler, less parameters for updateDOM

Reply:
Ok, trust your judgment here, thanks!